src/pages/gen2/reference/cli-commands/index.mdx

```diff
@@ -215,9 +215,81 @@ npx amplify generate config --stack amplify-nextamplifygen2-josef-sandbox-ca85e1
 
 ## npx amplify generate graphql-client-code
 
-Generate GraphQL statements for your frontend application to consume.
+Generate GraphQL statements and types for your frontend application to consume. 
+
+<SubCommandHeading parentCommand="amplify-graphql-client-code">Options</SubCommandHeading>
```

Comment:
this subheading's anchor should match the command name (e.g. `amplify-generate-graphql-client-code`)